index.js

```diff
@@ -0,0 +1,26 @@
+import Fastify from 'fastify'
+const fastify = Fastify({
+  logger: true
+})
+
+// Declare a route
+fastify.get('/', function (request, reply) {
+    reply.send({ hello: 'world' })
+})
+
+fastify.get('/test', function (request, reply) {
+    reply.send({hello: 'cct'});
+})
+
+fastify.get('/sum', function (request, reply) {
+    const a = parseInt(request.query['a']);
+    const b = parseInt(request.query['b']);
```

Comment:
a and b might not necessarily be integers. You can call the service as `localhost:3000/sum?a=bananas&b=3`